app/src/sharedTest/java/org/oppia/app/walkthrough/WalkthroughActivityTest.kt

```diff
@@ -72,17 +72,15 @@ class WalkthroughActivityTest {
     }
   }
 
-  /* ktlint-disable max-line-length */
```

Comment:
If you going to add documentation in wiki, you need to revert this line.

Reply:
It's already removed in the latest commit.

Reply:
So, we need it in this PR or no?

Reply:
we have 100 char limit check but I am not using `/* ktlint-disable max-line-length */` as this will check a complete block of line, rather than I am using `// ktlint.....` a single comment in whichever line is extending more than 100 char.

Reply:
Oh for that reason you removed it. I thought as you didnt add kdoc you removed it.

Reply:
No no, I just update in a cleaner way, all details will add in wiki after everything gets merged.

Reply:
Great thanks. Good to go.